components/vr-model.js

```diff
@@ -12,22 +12,21 @@
 
               var self = this;
               var src = this.getAttribute('src');
-              var scale = parseFloat(this.getAttribute('scale')) || 1;
+              
+              // The default THREE.ColladaLoader scale of 0.01 ensures scales match across DAE and THREE scene (eg 1m - 1m)
+              var scaleBase = 0.01;
+              var scaleUser = parseFloat(this.getAttribute('scale')) || 1;
+              var scale = scaleBase * scaleUser;
 
               // TODO: enable user to pass in material, and have that material apply to all nodes in the loaded object.
-
               // TODO: load and playback animations from loaded models.
-
-              // TODO: figure out lighting model. 
-              // self.scene.add( new THREE.AmbientLight( 0xcccccc ) ); 
-
+             
               var loader = new THREE.ColladaLoader();
               // loader.options.convertUpAxis = true; // Not sure if we need this. Doesn't appear to be the case. But it was in Three.js examples.
               loader.load( src, function ( collada ) {
                 var dae = collada.scene;
-                console.log(dae);
-                // dae.scale.x = dae.scale.y = dae.scale.z = scale;
-                // dae.updateMatrix();
+                dae.scale.x = dae.scale.y = dae.scale.z = scale;
+                dae.updateMatrix();
```

Comment:
You should do ```scale.set(scale, scale, scale)``` and no need to update the matrix here. It will be done for you in the rAF loop